src/scriptworker/utils.py

```diff
@@ -25,6 +25,7 @@
 import yaml
 from taskcluster.client import createTemporaryCredentials
 
+from scriptworker.context import Context
```

Comment:
This causes a circular import.

Reply:
@escapewindow circular import is now fixed, I opted for a "Don't refactor" solution